README.md

```diff
@@ -33,7 +33,7 @@
 </p>
 
 
-# Installation (CocoaPods)
+## Installation (CocoaPods)
```

Comment:
I don't get it, I thought that `<h3 align="center">Demo application on the App Store</h3>` is simply a part of the custom README header. What would you like to change the `<h3>` to then?

Reply:
As agreed, I changed the `<h3>` to `<p>` and used `<b>` to add emphasis, so that now GitHub won't interpret the `Demo application on the App Store` as a separate section when generating table of contents.